src/templates/Studio/Studio.js

```diff
@@ -55,17 +56,22 @@ export const query = graphql`
   }
 `;
 
-export default ({ location, data: { indexPage, studioPage } }) => (
-  <Layout>
-    <SEO title="Studio" pathName={studioPage.fields.slug} />
-    <Intro
-      data={{ ...indexPage.frontmatter, ...studioPage.frontmatter }}
-      location={location}
-    />
-    {studioPage.frontmatter.infoBlock.map(({ collection }, index) => (
-      <InfoBlock key={index} collection={collection} />
-    ))}
-    <Impression data={studioPage.frontmatter.studioImpression} />
-    <Footer {...indexPage.frontmatter} />
-  </Layout>
-);
+export default ({ location, data: { indexPage, studioPage } }) => {
+  const introRef = useRef();
+  return (
+    <Layout globalClass="studio">
```

Comment:
Not sure what the `"studio"` globalClass actually does. Is it somewhere in css? Maybe I just missed it.